client/my-sites/plans/current-plan/index.jsx

```diff
@@ -69,6 +77,21 @@ class CurrentPlan extends Component {
 				' its professional features.' );
 		}
 
+		if ( plan === PLAN_JETPACK_PERSONAL || plan === PLAN_JETPACK_PERSONAL_MONTHLY ) {
```

Comment:
We totally need to move these taglines to `lib/plans/constants` to avoid the repetitive logic here.